api/src/main/thrift/org/apache/aurora/gen/api.thrift

```diff
@@ -238,6 +238,10 @@ union Resource {
   5: i64 numGpus
 }
 
+struct KillPolicy {
+  1: i64 gracePeriod   /** Seconds */
```

Comment:
include seconds in the name, the API should be descriptive on what it's expecting (use something  like gracePeriodSecs). 
Add the doc above describing what's this.